asistente_ladm_col/logic/quality_rules/qr_plot_nodes_covered_boundary_points.py

```diff
@@ -0,0 +1,118 @@
+"""
+/***************************************************************************
+                              Asistente LADM-COL
+                             --------------------
+        begin           : 2022-06-20
+        git sha         : :%H$
+        copyright       : (C) 2021 by Germán Carrillo (SwissTierras Colombia)
+                          (C) 2022 by Sergio Ramírez (SwissTierras Colombia)
+        email           : gcarrillo@linuxmail.org
+                          sramirez@colsolutions.com
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtCore import QCoreApplication
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import (QUALITY_RULE_LADM_COL_LAYERS,
+                                                   QUALITY_RULE_ADJUSTED_LAYERS,
+                                                   ADJUSTED_INPUT_LAYER,
+                                                   ADJUSTED_REFERENCE_LAYER)
+from asistente_ladm_col.config.layer_config import LADMNames
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR3008,
+                                                           QRE_IGACR3008E01)
+from asistente_ladm_col.core.quality_rules.abstract_polygon_quality_rule import AbstractPolygonQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.lib.geometry import GeometryUtils
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRPlotNodesCoveredBoundaryPoints(AbstractPolygonQualityRule):
```

Comment:
This quality rule must be inherited from  AbstractPointQualityRule because the geometry of the error is a point

Reply:
Fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/commit/5d42252b237e39c64637f2aa4d703fe384e38250